.github/workflows/feature-close.yml

```
@@ -0,0 +1,40 @@
+          fetch-depth: 0
+
+      - name: Prepare feature deployment tag
+        uses: ./.github/actions/feature-checkout
+        id: prepare-docker-tag
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_webhook_url: ${{ secrets.SLACK_BUILD_ISSUES_REPORTING_WEBHOOK_URL }}
+
+      - name: Dispatch workflow to helm-values repository
+        env:
+          HELM_VALUES_TOKEN: ${{ steps.get-token.outputs.token }}
```

actionlint (1.7.4)

property "get-token" is not defined in object type {prepare-docker-tag: {conclusion: string; outcome: string; outputs: {feature_name: string; git_branch: string; git_branch_deploy: string}}}

(expression)